src/ripple/app/misc/NegativeUNLVote.cpp

```diff
@@ -198,7 +198,7 @@ NegativeUNLVote::buildScoreTable(
     for (int i = 0; i < FLAG_LEDGER_INTERVAL; ++i)
     {
         for (auto const& v : validations.getTrustedForLedger(
-                 ledgerAncestors[numAncestors - 1 - i]))
+                 ledgerAncestors[numAncestors - 1 - i], seq - 2 - i))
```

Comment:
There should also be a Negative UNL test to see what it does with a bogus sequence in a validation, independent from the other tests.